migration/migrator/migrations/course/20200529001923_cgi_tmp_folder.py

```diff
@@ -0,0 +1,22 @@
+import os
+import grp
+from pathlib import Path
+
+
+def up(config, database, semester, course):
+    tmp_cgi_dir = Path(config.submitty['submitty_data_dir'], 'tmp', 'cgi')
+
+    # create the directories
+    os.makedirs(str(tmp_cgi_dir), exist_ok=True)
+
+    php_user = config.submitty_users['php_user']
+    daemoncgi_group = config.submitty_users['daemoncgi_group']
+
+    # set the owner/group/permissions
+    chown_res = subprocess.check_output(["chown", "-R", php_user + ":" + daemoncgi_group + " ", str(tmp_cgi_dir)])
+    chmod_user_res = subprocess.check_output(["chmod", "-R", "u+rwx", str(tmp_cgi_dir)])
+    chmod_group_res = subprocess.check_output(["chmod", "-R", "g+rwxs", str(tmp_cgi_dir)])
+    chmod_other_res = subprocess.check_output(["chmod", "-R", "o-rwx", str(tmp_cgi_dir)])
```

Comment:
should these be assigned to variables or I should just simply use `subprocess.checkout`?

Reply:
If we're not using the variables, then no point in assigning them to anything. In this case, we're using this function as if the command fails, it'll throw an exception, indicating failure, which is what we want.

Reply:
I thought we are going to use those variables in some way for checking exceptions...but it seems it is not the case :) thanks!